client/web/src/backend/graphql.ts

```diff
@@ -15,6 +15,10 @@ const getHeaders = (): { [header: string]: string } => {
     if (trace) {
         headers['X-Sourcegraph-Should-Trace'] = trace
     }
+    const feat = parameters.getAll('feat')
+    if (feat) {
```

Comment:
`URLSearchParams.getAll()` [returns an array](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/getAll#return_value) that will always be truthy here. To avoid setting the header when it's redundant, we can check for an array length:

```suggestion
    const feat = parameters.getAll('feat')
    if (feat.length) {
```